### ISSUE TITLE
Doesn't start on Google Cloud

### ISSUE THREAD

**Issue by haf** [kubernetes/heapster#1303]
Google cloud reports Heapster running when you set up a new cluster, but it cannot be accessed at the reported uri: `Heapster is running at https://104.155.69.50/api/v1/proxy/namespaces/kube-system/services/heapster`

Reports `404 Page Not Found`.

Following the docs doesn't help; following them causes conflicts.

**Comment by piosz** [comment]
This is working as intended. There is nothing to display at this level of the path. Try to query for something more concrete, for example take a look into https://github.com/kubernetes/heapster/blob/master/docs/model.md

**Comment by haf** [comment]
Makes sense. What confuses me is whether there's a GUI anywhere which I as a human can use to peruse (pun intended) the metrics? The `grafana` folder begs the question.

Also, that your quickstart causes conflicts on GCE might be an issue?

**Comment by piosz** [comment]
The are two GUI-s showing metrics which comes out of the box on GCE: Grafana and [dashboard](https://github.com/kubernetes/dashboard).

Grafana is showing graphs based on data in InfluxDB, where data are being exported by Heaspter.
Dashboard is directly querying Heapster using model api.

**Comment by haf** [comment]
I'm sorry, but even googling for 15 minutes doesn't make me find where I access the grafana dashboard. And what about the quickstart?

I did find https://github.com/kubernetes/kubernetes/issues/4721 but there's nothing at `/api/v1/proxy/namespaces/kube-system/services/monitoring-grafana`

**Comment by piosz** [comment]
If you are creating your cluster with kube-up.sh, once your cluster is setup you can see a list of avaialble services, something like this:
```
kubernetes-dashboard is running at https://<master-ip>/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
Grafana is running at https://<master-ip>/api/v1/proxy/namespaces/kube-system/services/monitoring-grafana
```
This can be printed with `kubectl cluster-info` command.

Please have in mind that if you are using GKE there is no Grafana by default.

**Comment by haf** [comment]
I'm using Google Container Engine and would like to use this with it... Is there a guide for that?

**Comment by piosz** [comment]
Unfortunately there is no such guide.

The suggested options for displaying metrics on GKE are:
- dashboard as described above, more info here: https://cloud.google.com/container-engine/docs/oss-ui
- [Stackdriver](https://app.google.stackdriver.com/)
Both works out of the box in your cluster (unless you disabled monitoring).

If you really, really want to have Grafana in your cluster you can try to create monitoring pipeline on your own with heapster exporting to influxdb and grafana reading from it. You may want to use [cluster/addons/cluster-monitoring](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons/cluster-monitoring/influxdb) manifests. You should change the namespace to avoid conflict with existing monitoring pipeline. Also please have in mind that [heapster-controller.yaml](https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/cluster-monitoring/influxdb/heapster-controller.yaml) is a template and you should evaluate it (probably manually).

To sum up the procedure should be:
1. change namespace from kube-system to another one
2. evaluate Heapster manifest
3. create all resources form the directory
